Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM python:3.10
+FROM python:3.10-alpine3.18
```

Comment:
I prefer to use the [python:3.10-alpine](https://hub.docker.com/layers/library/python/3.10-alpine/images/sha256-e1d7fc61bb8a17c5a366c7a63c156e0ba1215e4d0dcee71ee85010be64bc51a0?context=explore) as it's the latest and the project it's not pinned to the alpine3.18 version